public/css/style.css

```diff
@@ -0,0 +1,389 @@
+@import url('https://fonts.googleapis.com/css2?family=Comfortaa:wght@400;700&display=swap');
+
```

Comment:
Breaking up long css files could improve organization of the code. Using `@import ('./another.css');` and turning `styles.css` into an aggregator for all your css files. Not necessary but I thought I'd mention it :) I use it [here](https://github.com/Llourn/tech-blog/blob/main/public/css/style.css) if you want to see an example of what I mean.